src/main/java/org/janelia/saalfeldlab/paintera/viewer3d/Scene3DHandler.java

```diff
@@ -86,53 +87,49 @@ public void setInitialTransformToInterval(final Interval interval)
 		InvokeOnJavaFXApplicationThread.invoke(() -> this.setAffine(initialTransform));
 	}
 
-	private void addCommands()
-	{
-		viewer.addEventHandler(ScrollEvent.SCROLL, event -> {
+  private void addCommands() {
 
-			final double scroll = event.getDeltaY();
-			double scrollFactor = scroll > 0 ? 1.05 : 1 / 1.05;
+	viewer.addEventHandler(ScrollEvent.SCROLL, event -> {
 
-			if (event.isShiftDown())
-			{
-				if (event.isControlDown())
-				{
-					scrollFactor = scroll > 0 ? 1.01 : 1 / 1.01;
-				}
-				else
-				{
-					scrollFactor = scroll > 0 ? 2.05 : 1 / 2.05;
-				}
-			}
 
-			if (Math.abs(event.getDeltaY()) > Math.abs(event.getDeltaX()))
-			{
-				final Affine target = affine.clone();
-				target.prependScale(scrollFactor, scrollFactor, scrollFactor);
-				InvokeOnJavaFXApplicationThread.invoke(() -> this.setAffine(target));
+	  final double scroll = ControlUtils.getBiggestScroll(event);
```

Comment:
indentation?

Reply:
This one's on me, since it's actual new code ( this is the Shift+Scroll update). I'll make this consistent with the rest of the file.

Reply:
I've updated the formatting